### PR TITLE
(SUP-2500) Remove Harmful Technology - Readme and Reference

### PR DESCRIPTION
**Pull request by henrywangpuppet**
Change the harmful terminology per:

blacklist > blocklist
compile master > compiler
high availability > disaster recovery
master of masters > primary server
master > primary server
master (branch) > main (branch)
monolithic > standard (but you might mean all supported architectures: https://support.puppet.com/hc/en-us/articles/360037742154-Architecture-terminology-changes-in-Puppet-Enterprise-2019-2)
whitelist > allowlist

Most of the terms discovered and changed were `master` -> `primary server` and `masters` -> `primary server and compilers`

### CODE REVIEW

**Review by asselvakumar — APPROVED** [review]
Changes are looking good to merge.

File: README.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ You have the option of collecting metrics using any or all of the following meth
 ## Setup
 
 > In PuppetDB 6.9.1 & 5.2.13 and newer, the `/metrics/v1` endpoints are disabled by default and access to the `/metrics/v2` endpoints are restricted to localhost only in response to [CVE-2020-7943](https://nvd.nist.gov/vuln/detail/CVE-2020-7943). 
-Starting with version 2.3.0 of this module, PuppetDB metrics will not be setup by the main class if you are on the versions above or higher unless the main class is applied to the master. To collect PuppetDB metrics in other scenarios, you should use the `puppet_metrics_dashboard::profile::puppetdb` class applied to any PuppetDB nodes with the option `enable_client_cert => false` (the request will be to localhost and doen't require SSL)
+Starting with version 2.3.0 of this module, PuppetDB metrics will not be setup by the main class if you are on the versions above or higher unless the main class is applied to the primary server. To collect PuppetDB metrics in other scenarios, you should use the `puppet_metrics_dashboard::profile::puppetdb` class applied to any PuppetDB nodes with the option `enable_client_cert => false` (the request will be to localhost and doen't require SSL)
```
Comment (MartyEwings) [inline README.md:45]:
```suggestion
Starting with version 2.3.0 of this module, PuppetDB metrics will not be setup by the main class if you are on the versions above or higher unless the main class is applied to the Primary Server. To collect PuppetDB metrics in other scenarios, you should use the `puppet_metrics_dashboard::profile::puppetdb` class applied to any PuppetDB nodes with the option `enable_client_cert => false` (the request will be to localhost and doen't require SSL)
```

File: REFERENCE.md [hunk 2]
```diff
@@ -34,7 +34,7 @@
 ### Defined types
 
 * [`puppet_metrics_dashboard::certs`](#puppet_metrics_dashboardcerts): This class creates a certificates for Grafana and for connecting to PE Postgres.
-* [`puppet_metrics_dashboard::profile::compiler`](#puppet_metrics_dashboardprofilecompiler): Apply this class to a master or compiler to collect puppetserver metrics
+* [`puppet_metrics_dashboard::profile::compiler`](#puppet_metrics_dashboardprofilecompiler): Apply this class to a primary server or compiler to collect puppetserver metrics
```
Comment (MartyEwings) [inline REFERENCE.md:37]:
```suggestion
* [`puppet_metrics_dashboard::profile::compiler`](#puppet_metrics_dashboardprofilecompiler): Apply this class to a Primary Server or Compiler to collect puppetserver metrics
```

File: REFERENCE.md [hunk 3]
```diff
@@ -421,7 +421,7 @@ Default value: `lookup('puppet_metrics_dashboard::http_response_timeout')`
 
 Data type: `Variant[String,Tuple[String, Integer]]`
 
-The FQDN of the compiler / master.  Defaults to the FQDN of the server where the profile is applied
+The FQDN of the compiler / primary server.  Defaults to the FQDN of the server where the profile is applied
```
Comment (MartyEwings) [inline REFERENCE.md:424]:
```suggestion
The FQDN of the Compiler / Primary Server.  Defaults to the FQDN of the server where the profile is applied
```

**Review by MartyEwings — CHANGES_REQUESTED** [review]
There is a few places with inconsistent casing, I would change all instances of lower case compiler and primary server to uppercase

**Review by MartyEwings — APPROVED** [review]

**Comment by henrywangpuppet** [comment]
Closed due to a chaos

**Pull request closed without merging**